azavea/cac-tripplanner-android
# Feature/kak/add geofencing#44

## Overview

Add geofences when user flags a place or an event with an associated destination as "want to go". Remove geofences when flag changes to something else. Add geofences back after reboot. Send a notification when user enters a geofenced area. Notification links in to the detail page for the place or event.

## Demo

![screenshot_1528172229](https://user-images.githubusercontent.com/960264/40981310-a3a42f6a-68a8-11e8-8e5c-5b47b0bacab8.png)

## Notes

Device location settings must be set to "high accuracy" for geofence events to fire.

## Testing

 - On app launch, system settings for location will open if "high accuracy" not set (emulator default is "device only")
- Flag a place or event "want to go"
- Send a location to the emulator within 300 meters of the place
- Open Maps to cause the device to request location
- Location may need to be sent repeatedly for device location to move to place
- Should receive notification that links to detail page

Closes #44 
Closes #45
